docs/release-notes/8.11.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
```

Comment:
@nastasha-solomon do we want to add a known issue section here\
```
[discrete]
[[known-issue-8.12.0]]
==== Known issues
```
and add the TIP issue [I just created](https://github.com/elastic/security-docs/issues/4582)? The issue appeared between `8.11.0` and 8.11.1`

Reply:
Added this in [f8f427e](https://github.com/elastic/security-docs/pull/4547/commits/f8f427e2a841ca0b8ed00b82e0d04a6e016d29cb) and [9715150](https://github.com/elastic/security-docs/pull/4547/commits/9715150966576f3994bb8f3c2e8b632e825f180d)